README.md

```diff
@@ -135,6 +135,12 @@ trufflehog filesystem path/to/file1.txt path/to/file2.txt path/to/dir
 trufflehog gcs --project-id=<project-ID> --cloud-environment --only-verified
 ```
 
+# 8: Scan a Docker image for verified secrets.
+
+```bash
+trufflehog docker --image trufflesecurity/secrets --only-verified
```

Comment:
**suggestion** (non-blocking): might be worth mentioning you can pass multiple images to scan here

Reply:
Added